pkg/runtime/runtime.go

```diff
@@ -140,8 +140,11 @@ type Route struct {
 	rules    []*runtime_pubsub.Rule
 }
 
+type TopicRoutes map[string]TopicRoute
```

Comment:
nitpicking:

Generally speaking, we read the code from top lines to bottom lines, given that this line is using the TopicRoute type (that's defined below) I think it should be better placed right after the `TopicRoute` type,

just nitpicking

Reply:
Yeah on this and the next comment: I agree naming can be improved. I'll change